integration-tests/basic-test-app/helpers/generateSpans.ts

```diff
@@ -0,0 +1,83 @@
+/* 'use strict'; */
+/* eslint-disable no-unused-vars */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import {
+  context,
+  SpanKind,
+  SpanStatusCode,
+  trace,
+  Tracer,
+} from "@opentelemetry/api";
+
+export function generateTestSpans(tracer: Tracer) {
+  const span1 = tracer.startSpan("test-1");
+  const span2 = tracer.startSpan("test-2", { kind: SpanKind.CLIENT });
+  const span3 = tracer.startSpan("test-3");
+  // Not ended so shouldn't be part of the output
```

Comment:
they should be independent the idea would be to call generateTestSpans followed by ending the session so the payload can be verified then separately in a following test we'll call generateNestedSpans and end that session so the payload shouldn't include data from the previous run

Reply:
Ah OK. That makes sense. Is that documented or implicit anywhere that I missed?

Reply:
let me add some comments to this file, I wasn't sure if I was going to stick with this approach as there were 2 options:
1. in the basic test app expose buttons/inputs for interacting with the spans API so that the test case can do all the individual setup (e.g. start span, add attributes, etc.)
2. in the basic test app expose a single "test this scenario" button and do the asserts in the test case

1) I think is ideal because then you can read all the setup and assertions together in the test, however when I started to implement this way I found it was just very time consuming to create the full interface that was needed to do all the setup that is being done here

2) simplifies things but creates this disconnect between the assertions in the test spec and what setup was actually done, probably going to stick with this for now but I'll add comments to this file to make it clear why that was separated